app/src/main/java/com/example/keybindhelper/ui/keybind/KeybindFragment.kt

```diff
@@ -19,79 +19,80 @@ import com.example.keybindhelper.dao.CurrentProjectManager
+        val rv = view.findViewById<RecyclerView>(R.id.recyclerView)
 
-        mainActivity.Menu!!.findItem(R.id.action_delete_all_groups).setOnMenuItemClickListener {
-            val cd=ConfirmDialog(view.context,"Delete All Groups?")
-            cd.onConfirmed = object : ConfirmDialog.ConfirmedEvent{
+        mainActivity.Menu.findItem(R.id.action_delete_all_groups).setOnMenuItemClickListener {
+            val cd = ConfirmDialog(view.context, "Delete All Groups?")
+            cd.onConfirmed = object : ConfirmDialog.ConfirmedEvent {
                 override fun onConfirmed() {
                     CurrentProjectManager.deleteAllGroups()
                     rv.adapter = GroupAdapter(CurrentProjectManager.Groups)
                 }
             }
             cd.Show()
 
-            true;
+            true
         }
-        mainActivity.Menu!!.findItem(R.id.action_sub_hide_keybinds).setOnMenuItemClickListener {
-            for(v in rv.children)
-                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = false;
-            true;
+        mainActivity.Menu.findItem(R.id.action_sub_hide_keybinds).setOnMenuItemClickListener {
+            for (v in rv.children)
+                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = false
+            true
         }
-        mainActivity.Menu!!.findItem(R.id.action_sub_show_keybinds).setOnMenuItemClickListener {
-            for(v in rv.children)
-                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible=true;
-            true;
+        mainActivity.Menu.findItem(R.id.action_sub_show_keybinds).setOnMenuItemClickListener {
+            for (v in rv.children)
+                v.findViewById<RecyclerView>(R.id.keybind_zone).isVisible = true
+            true
         }
-        mainActivity.Menu!!.findItem(R.id.action_add).setOnMenuItemClickListener {
+        mainActivity.Menu.findItem(R.id.action_add).setOnMenuItemClickListener {
             CurrentProjectManager.AddGroup()
-            System.out.println("MainActivity.floatingactionbutton.click: Groups Size:" + CurrentProjectManager.Groups!!.size)
+            println("MainActivity.floatingactionbutton.click: Groups Size:" + CurrentProjectManager.Groups!!.size)
             rv.adapter!!.notifyItemChanged(CurrentProjectManager.Groups!!.size - 1)
-            true;
+            true
         }
     }
 
     private fun loadRecycleView(root: View) {
-        val rv = root.findViewById<RecyclerView>(R.id.recyclerView);
-        rv!!.adapter = GroupAdapter(CurrentProjectManager.Groups);
-        rv.layoutManager = LinearLayoutManager(root.context);
+        val rv = root.findViewById<RecyclerView>(R.id.recyclerView)
+        rv!!.adapter = GroupAdapter(CurrentProjectManager.Groups)
+        rv.layoutManager = LinearLayoutManager(root.context)
     }
 
 
 }
+
+private fun Any.setOnMenuItemClickListener(function: () -> Boolean) {
+
+}
+
+private fun Any.findItem(actionDeleteAllGroups: Any) {
+
+}
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.